examples/mlperf/scripts/setup_kits19_dataset.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
@chenyuxyz - thoughts on making this entire script `tinybox` specific? that is, after downloading the dataset, it moves it to the `/raid` drive and symbolically links the dataset to the user's `extra/datasets/kits19`?

Reply:
ideally it should detect if the dataset already exists and only download if needed. i think how resnet/bert handle it is fine (pass in a BASE_DIR for the data, and in the main script do some sanity check of the data in that dir)

Reply:
i updated the shell script now to do some basic validation and use a `BASEDIR` environment variable as well.